cadasta/plugin.py

```diff
@@ -189,7 +180,7 @@ def unload(self):
         # remove the toolbar
         self.iface.mainWindow().removeToolBar(self.toolbar)
 
-    def _create_options_wizard_action(self):
+    def _create_options_wizard(self):
         """Create action for options wizard."""
         icon_path = ':/plugins/cadasta-qgis-plugin/icon.png'
```

Comment:
hi @Gustry , it is fixed in another PR: https://github.com/kartoza/cadasta-qgis-plugin/pull/52